ros_foxglove_msgs/ros1/CameraCalibration.msg

```diff
@@ -1,4 +1,5 @@
-# Generated from CameraCalibration by @foxglove/schemas
+# foxglove_msgs/CameraCalibration
```

Comment:
I'd like to advocate for keeping the `generated from` text but having it reference he repository. `@foxglove/schemas` is an npm specific name and wouldn't help you figure out where the code was that generated these files.

I do not think having `foxglove_msgs/CameraCalibration` is adding anything as a comment.

Reply:
If you want to make it clear that it belongs to a ros package, just be explicit and say `Part of the foxglove_msgs package` - even tho I'm not sure that matters since a user is gonna need to go find the install instructions for foxglove_msgs package anyway - maybe the comment can link there but then its kind of silly since how did you install those in the first place?

Reply:
I link people to this repo, sometimes directly to the individual messages. I would like it to be clear to the reader what the name of this package and schema is, since it is not otherwise mentioned within the file.